init-tools.sh

```diff
@@ -132,7 +132,7 @@ if [ ! -e $__DOTNET_PATH ]; then
         cd $__DOTNET_PATH
         tar -xf $__DOTNET_PATH/dotnet.tar
     }
-    execute installDotNetCLI >> "$__init_tools_log" 2>&1
+    executeWithRetry installDotNetCLI >> "$__init_tools_log" 2>&1
```

Comment:
I didn't notice this before with just the `installDotNetCLI` function--elsewhere in this file and `dotnet-install.sh` I see we typically use `_` separators rather than camelCase. I think we should stick to that.

Reply:
Fixed.